<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoConfigStreamPage</class>
 <widget class="QWidget" name="AutoConfigStreamPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Basic.AutoConfig.StreamPage.Service</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="service"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="regionLabel">
     <property name="text">
      <string>BandwidthTest.Region</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="region" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="regionUS">
        <property name="text">
         <string>BandwidthTest.Region.US</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="regionEU">
        <property name="text">
         <string>BandwidthTest.Region.EU</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="regionAsia">
        <property name="text">
         <string>BandwidthTest.Region.Asia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="regionOther">
        <property name="text">
         <string>BandwidthTest.Region.Other</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Basic.AutoConfig.StreamPage.StreamKey</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="show">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
